cli/src/pcluster/schemas/imagebuilder_schema.py

```diff
@@ -60,10 +60,11 @@ def validate_reserved_tag(self, value):
         """Validate reserved tag in tags."""
         if value:
             for tag in value:
-                if tag.key == "PclusterVersion":
+                match = re.match(r"^pcluster_*", tag.key)
+                if match:
                     raise ValidationError(
-                        message="The Key 'PclusterVersion' used in your 'tags' configuration parameter "
-                        "is a reserved one, please change it."
+                        message="The tag key names 'pcluster_*' are reserved tag key names, "
```

Comment:
The tag key prefix 'pcluster_' is reserved and cannot be used.

Reply:
done